README.md

```diff
@@ -87,6 +87,24 @@ sql, args, err := sq.Delete("a1", "a2").
 
 Package [pg](https://godoc.org/github.com/elgris/sqrl/pg) contains PostgreSQL specific operators.
 
+#### [Update from](https://www.postgresql.org/docs/current/static/sql-update.html)
+
+```go
+sql, args, err := sq.Update("a1").
+    Set("foo", 1).
+    From("a2").
+    Where("id = a2.ref_id AND a2.num = ?", 42).
+    ToSql()
+```
+
+#### [Delete using](https://www.postgresql.org/docs/current/static/sql-delete.html)
```

Comment:
Looks like `DELETE ... USING` is also supported by MySQL in [multi-table delete statement](https://dev.mysql.com/doc/refman/5.7/en/delete.html). Could you please add a piece of doc to MySQL section above? Additional test is also highly appreciated :)

Reply:
Test is implemented in `delete_test.go` - `TestDeleteUsing`

Reply:
Wonderful! Thanks!